data/tests/quirks.d/merged.quirk

```diff
@@ -0,0 +1,2 @@
+[USB\VID_0A5C&PID_6412]
+Flags = MERGE_ME
```

Comment:
Do you really want to be using `MERGE_ME`?  I would think using one of the existing flags like `is-bootloader` should be better.

Reply:
That's kinda the point, it's a nonstandard flag. I'll add some more work on this to make it all a bit clearer.